File: DownloadFiles/TGNet/Chapter03/C#/Exercise3-1/FirstTestStartWithThis/FirstTest/FirstTest/Form1.cs
#region Using directives

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Windows.Forms;
using System.IO;

#endregion

namespace FirstTest
{
    partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {

            if (File.Exists(textBox1.Text))
                MessageBox.Show("File Exists: Test passed");
            else
                MessageBox.Show("File doesn’t Exist: Test failure");
        }
    }
}

File: DownloadFiles/TGNet/Chapter03/VB/Exercise3-1/FirstTestStartWithThis/FirstTest/FirstTest/Form1.vb
Imports System.IO
Public Class Form1

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        If File.Exists(TextBox1.Text) Then
            MessageBox.Show("File Exists: Test passed")
        Else
            MessageBox.Show("File doesn't Exist: Test failure")
        End If

    End Sub
End Class
